src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrintingServices.Unix.cs

```diff
@@ -659,15 +659,15 @@ internal static void GetPrintDialogInfo(string printer, ref string port, ref str
                 if (!found)
                     return;
 
-                cups_dests = (CUPS_DESTS)Marshal.PtrToStructure(ptr_printers, typeof(CUPS_DESTS));
+                cups_dests = (CUPS_DESTS)Marshal.PtrToStructure(ptr_printers, typeof(CUPS_DESTS))!;
 
                 NameValueCollection options = LoadPrinterOptions(cups_dests.options, cups_dests.num_options);
 
                 if (options["printer-state"] != null)
-                    state = int.Parse(options["printer-state"]);
+                    state = int.Parse(options["printer-state"]!);
 
                 if (options["printer-comment"] != null)
-                    comment = options["printer-state"];
+                    comment = options["printer-state"]!;
```

Comment:
NIT: please add // TODO-NULLABLE https://github.com/dotnet/roslyn/issues/34644

Reply:
Looks very likely, but wouldn't want to include the fix in this work.

Reply:
cc @JeremyKuhne

Reply:
> Looks very likely, but wouldn't want to include the fix in this work.

Ok, not sure if we file and issue for this and https://github.com/dotnet/runtime/pull/31716#discussion_r380972902 for further action

Reply:
@eiriktsarpalis would you mind opening issues to track fixing these? I agree that we should do these separately.

Reply:
Done